playbooks/roles/ocp-verification-tests/tasks/main.yml

```diff
@@ -5,10 +5,22 @@
       - git
     state: latest
 
-- name: Install Ruby-3 package
-  shell: |
-    sudo dnf module reset ruby -y
-    sudo dnf install -y @ruby:3.0
+- name: Uninstall Ruby
+  ansible.builtin.dnf:
+    name: ruby
+    state: absent
+
+- name: Reset the Ruby module
+  shell: sudo dnf module reset ruby -y
+
+- name: Get latest stream for Ruby
+  shell: dnf module list ruby | grep ruby | awk 'END {print $2}'
```

Comment:
Can you please try to explain what this line does? I am confused you are trying to `list ruby` and again `grep ruby`. IMO this couple print multiple ruby streams, with the name `*ruby*` in it.

Reply:
When execute cmd `dnf module list ruby` , we will get multiple ruby streams, I am using grep to filter rows for ruby.  Then selecting the last stream.

```
#  dnf module list ruby
Updating Subscription Management repositories.
Last metadata expiration check: 2:46:42 ago on Thu 09 Feb 2023 12:58:36 AM EST.
Red Hat Enterprise Linux 8 for Power, little endian - AppStream (RPMs)
Name       Stream        Profiles            Summary
ruby       2.5 [d]       common [d]          An interpreter of object-oriented scripting language
ruby       2.6           common [d]          An interpreter of object-oriented scripting language
ruby       2.7           common [d]          An interpreter of object-oriented scripting language
ruby       3.0           common [d]          An interpreter of object-oriented scripting language
ruby       3.1 [e]       common [d] [i]      An interpreter of object-oriented scripting language

Hint: [d]efault, [e]nabled, [x]disabled, [i]nstalled
```

```
# dnf module list ruby | grep ruby | awk 'END {print $2}'
3.1
```